{{ .ProjectSnake }}/tools/oci/py3_image.bzl

```diff
@@ -30,7 +30,7 @@ def py3_image(name, binary, root = "/", layer_groups = {}, env = {}, workdir = N
         base = base,
         tars = py_image_layer(
             name = name + "_layers",
-            py_binary = binary,
+            binary = binary,
```

Comment:
The template already upgraded to the version with the breaking change?

Reply:
Yes I've been accepting renovate PRs here, it's quite current on the latest.